schema/link.yaml

```diff
@@ -6,6 +6,39 @@ description: WCMP2 link object definition
 allOf:
   - $ref: https://raw.githubusercontent.com/opengeospatial/ogcapi-records/master/core/openapi/schemas/common/link.yaml
 properties:
-  wmo:topic:
-    type: string
-    description: the PubSub subscription topic
+  distributionInformation:
+    type: object
+    description: the additional information qualifying the service and allowing to build and display the "access" information in a portal
+	properties: 
+      availableFormats: 
+        type: array 
+        items: 
+           type: object
+           description: 
+              type: string
+              description: the individual format description to allow building the portal information
+           numberOFiles: 
+              type: string
+              description: typically 300 per day or 4 per hour
```

Comment:
`description: typical number of files disseminated by the distribution mechanism over a period of time`